scikits/odes/sundials/cvode.pyx

```diff
@@ -257,15 +264,15 @@ cdef int _jacdense(long int Neq, realtype tt,
         jac_tmp = aux_data.jac_tmp
 
         nv_s2ndarray(yy, yy_tmp)
-    aux_data.jac.evaluate(tt, yy_tmp, jac_tmp)
+    user_flag = aux_data.jac.evaluate(tt, yy_tmp, jac_tmp)
 
     if parallel_implementation:
         raise NotImplemented
     else:
         #we convert the python jac_tmp array to DslMat of sundials
         ndarray2DlsMatd(Jac, jac_tmp)
 
-    return 0
+    return user_flag
```

Comment:
The conversion is always in `aux_data.<func>.evaluate`, where `<func>` is `rfn`, `rootfn`, `jac`, `prec_setupfn`, `prec_solvefn` or `jac_times_vecfn`. So the conversion has already been done there.